PeerioTechnologies/peerio-desktop
# Enable Windows updater

* Electron-updater removed 🎉 

* This PR also bumps peerio-updater version, which contains an unrelated fix for Linux updater (https://peerio.tpondemand.com/entity/7621). I though it would be okay to include it here, since it would be annoying to make separate PRs for that, and since both PRs would have to be tested via two staging releases.

### PR author

#### Relevant issue tracker links  

Not sure we have a task for it.

For testing on staging we have these:
https://peerio.tpondemand.com/entity/6163

#### Manual testing instructions  

It was already enabled on staging for some time, so we tested it. However, we should do the usual two-staging releases test:

1. Release staging.
2. Check that it launches.
3. Release another staging.
4. Check that the previous one updates to it.

Let's test all platforms.


----
### Reviewer

* Localisation.
* Architectural decisions.
* Performance.
* Comments.
* Big feature tech design documentation.
* Readability.
* DRY.
* Can we extract something to SDK?
* Are there any bicycles invented that should be replaced with existing SDK
* Are there unit and e2e tests added?

----
### Repository owner
